tests/Endpoints/ObjectiveTest.php

```diff
@@ -44,9 +50,9 @@ public function putsToTest()
             'position' => ['position', $this->getFaker()->randomDigit],
             'notActive' => ['active', false],
             'competency' => ['competency', 1],
-            'courseObjectives' => ['courseObjectives', [1], $skipped = true],
-            'programYearObjectives' => ['programYearObjectives', [2], $skipped = true],
-            'sessionObjectives' => ['sessionObjectives', [2], $skipped = true],
+//            'courseObjectives' => ['courseObjectives', [2]],
```

Comment:
I vote to remove these commented lines in favor of the test just working below.

Reply:
removed.